includes/discovery/sensors/temperature/timos.inc.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+if ($device['os'] === 'timos') {
+    $oids = snmp_walk($device, 'tmnxHwID', '-Osqn', 'TIMETRA-CHASSIS-MIB', 'aos');
```

Comment:
Quite possibly; but the context and use cases of the derivatives aren't made very clear.